src/finit.h

```diff
@@ -36,6 +36,16 @@
 #include <lite/lite.h>
 #include <uev/uev.h>
 
+/* just in case */
+#ifndef _PATH_STDPATH
+#define _PATH_STDPATH   "/usr/bin:/bin:/usr/sbin:/sbin"
+#endif
+
+#ifndef _PATH_VARRUN
+#define _PATH_VARRUN    "/var/run/"
```

Comment:
This is fine, but I think most distributions are moving towards `/run`, so that they can keep it on a RAM disk.